remote/remote_state_s3.go

```diff
@@ -313,6 +316,10 @@ func parseExtendedS3Config(config map[string]interface{}) (*ExtendedRemoteStateC
 		return nil, errors.WithStackTrace(err)
 	}
 
+	if config["accesslogging_target_prefix"] == "" {
```

Comment:
Is empty string here the same as not being set at all?

Or do you actually want:

```go
prefix, prefixIsSet := config["accesslogging_target_prefix"]
if !prefixIsSet {
  // If no prefix is set, we set a default one
  config["accesslogging_target_prefix"] = s3BucketAccessLoggingTargetPrefix
}
```

Reply:
ah shit - thought I marked this as WIP, as I was testing the different cases - so this is still WIP - very much thanks for the pointers, Jim!